patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Bud Gidiere <sgidiere@gmail.com>
+Date: Mon, 10 Aug 2020 14:11:47 -0500
+Subject: [PATCH] IntelliJ Remove Streams
+
+Do not update, just regen.
+
+diff --git a/src/main/java/net/minecraft/server/BehaviorGate.java b/src/main/java/net/minecraft/server/BehaviorGate.java
+index 46e910581210421c8699637431804dc2f43eb4a6..945c7d0957e5fcdd4dc3b3ac95e274bdee23bf3f 100644
+--- a/src/main/java/net/minecraft/server/BehaviorGate.java
++++ b/src/main/java/net/minecraft/server/BehaviorGate.java
+@@ -19,9 +19,9 @@ public class BehaviorGate<E extends EntityLiving> extends Behavior<E> {
+         this.b = set;
+         this.c = behaviorgate_order;
+         this.d = behaviorgate_execution;
+-        list.forEach((pair) -> {
```

Comment:
This isn't even a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/ArrayList.java#l1226